File: uva/java/uva_12577/src/uva_12577/Uva_12577.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package uva_12577;

/**
 *
 * @author DELL
 */
import java.util.*;
public class Uva_12577 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        Scanner scan = new Scanner(System.in);
        
            long i;
    
String in;
    i=1;
    while(true)
    {
        in = scan.nextLine();
                
        char input [] = in.toCharArray();

        if( input[0] == '*' )
        {
            break;
        }
            else if(input[0] == 'H')
            {
                System.out.println("Case "+i+": Hajj-e-Akbar");
                i++;
            }
                else
                {
                    System.out.println("Case "+i+": Hajj-e-Asghar");
                    i++;
                }
    }

    }
    
}

File: uva/c_cpp/uva-12577.c
#include <stdio.h>

int main()
{
    long long i;
    char input[100];

    i=1;
    while(1)
    {
        gets(input);

        if( input[0] == '*' )
        {
            break;
        }
            else if(input[0] == 'H')
            {
                printf("Case %lld: Hajj-e-Akbar\n",i);
                i++;
            }
                else
                {
                    printf("Case %lld: Hajj-e-Asghar\n",i);
                    i++;
                }
    }

    return 0;
}
